src/sage/rings/padics/pow_computer_ext.pyx

```diff
@@ -917,7 +918,7 @@ cdef class PowComputer_ZZ_pX(PowComputer_ext):
         EXAMPLES::
 
             sage: PC = PowComputer_ext_maker(5, 5, 10, 20, False, ntl.ZZ_pX([-5,0,1],5^10), 'FM', 'e',ntl.ZZ_pX([1],5^10))
-            sage: PC._restore_context_capdiv_test(8) #indirect doctest
+            sage: PC._restore_context_capdiv_test(8)  #indirect doctest
```

Comment:
space after `#`

Reply:
Done throughout sage.rings.padics in b28f6265550